src/tools/ceph_objectstore_tool.cc

```diff
@@ -646,23 +646,28 @@ int do_trim_pg_log(ObjectStore *store, const coll_t &coll,
 
   cerr << "Log bounds are: " << "(" << info.log_tail << ","
        << info.last_update << "]" << std::endl;
-
-  uint64_t max_entries = g_ceph_context->_conf->osd_max_pg_log_entries;
+  uint64_t max_entries = std::min<uint64_t>(g_ceph_context->_conf->osd_max_pg_log_entrie,
+					    g_ceph_context->_conf->osd_pg_log_trim_max);
```

Comment:
I don't understand this. `osd_max_pg_log_entrie` is how many entries we went to keep and `osd_pg_log_trim_max` is how many entries we want to trim in one iteration in `while (!done)` loop below. How can you mix this in one variable?